changelog.md

```diff
@@ -28,6 +28,9 @@
 - New feature: Add configuration option disconnect_on_unauthorized_publish_v3 to force disconnect on unauthorized publish even for MQTT clients before v3.1.1
 - New feature: Add persistent message queueing functionality to `vmq_bridge` Plugin, using [ReplayQ](https://github.com/emqx/replayq)
 
+## VerneMQ 1.13.0.1
```

Comment:
Please do not introduce non-existing tags/releases. Just add the line as the first line in changelog, without a release. Thanks, will review the rest later!